### DISCUSSION TITLE
Dose libhv support  Server-Sent Events protocol?

### DISCUSSION THREAD

**Discussion by qZhang88** [ithewei/libhv#471] [Q&A]
I want to write a C++ openai API to streaming LLM generated result. Since most GPT like streaming API use SSE protocal. Does libhv support it? And any example?

**Comment by ithewei** [comment]
https://github.com/ithewei/libhv/blob/dd2bb958f9ef41265c66c27d3ff499993a634812/examples/httpd/handler.cpp#L364

**Reply by qZhang88** [reply]
this seems to be server side sending event, any example for SSE client handle?

**Comment by ithewei** [comment]
> this seems to be server side sending event, any example for SSE client handle?

Try to set HttpRequest::http_cb to recv data.
